index.md

```diff
@@ -57,7 +57,7 @@ A practice is included in this minimal list if:
     *   And don't duplicate contents of stable, long-lived repositories (i.e., don't clone GenBank)
 2.  All synthesized data is stored in well-defined widely-used formats:
     *   CSV or HDF5 for tabular data
-    *   JSON or YAML for referential data
+    *   JSON, YAML, or XML for referential data, i.e. data that does not naturally form a rectangle
```

Comment:
Yes those are good and mean more to me than "referential".

I also talk about this a lot as "nested lists" but I have lost all perspective. Would that mean anything to other people?

"Not a natural rectangle" is the most important bit, I suspect.